src/beta_inc.jl

```diff
@@ -934,7 +934,6 @@ function _beta_inc_inv(a::Float64, b::Float64, p::Float64, q::Float64=1-p)
     r = sqrt(-2*log(p))
     p_approx = r - @horner(r, 2.30753e+00, 0.27061e+00) / @horner(r, 1.0, .99229e+00, .04481e+00)
     fpu = floatmin(Float64)
-    sae = log10(fpu)
```

Comment:
`sae` is only used in the computation of `acu = exp10(max(..., sae))` below. I rewrote this as `acu = max(exp10(...), fpu)` (or rather `10 * fpu`).